src/main/java/com/google/cloud/tools/appengine/cloudsdk/process/ProcessOutputLineListener.java

```diff
@@ -20,6 +20,7 @@
 
 /**
  * Line output listener for {@link DefaultProcessRunner}.
+ * Warning: all implementations must be thread safe.
```

Comment:
An explanation here would be useful: it took me a minute or two to puzzle through.  _The process writes lines, and any consumers listen from the lines._